tool/package.json

```diff
@@ -26,11 +26,12 @@
     "ledger-live": "./cli.js"
   },
   "files": [
+    "cli.js",
     "lib"
   ],
   "scripts": {
     "build": "babel --ignore __tests__ -sd lib src",
-    "prepublish": "yarn build",
+    "prepublishOnly": "rm -rf lib && babel --ignore __tests__ -d lib src",
```

Comment:
not sure about this change

Reply:
`prepublish` is run on publish _and_ on local install. It's getting deprecated by npm in favor of `prepublishOnly` which actually does what it says.

We don't really need to automatically build on `yarn install` nor we need babel maps on `yarn global add ledger-live`, so this builds only on publish, without babel's `-s` option.

This is just spit and polish, I don't mind reverting if you'd like.

Reply:
i'm fine keeping it if it works, right now it fails on the CI so that's why i asked (also you probably should factorize the command `yarn build`

Reply:
`master` fails on CI at the exact same point actually, so I doubt the issue comes from this change 😅

Reply:
(looks like `jest` fails since it's been updated a few commits ago)